assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -214,6 +214,47 @@ const AddressForm = ( {
 					);
 				}
 
+				const customValidationHandler = (
+					inputObject: HTMLInputElement
+				) => {
+					if (
+						! isPostcode( {
+							postcode: values.postcode,
+							country: values.country,
```

Comment:
Let's either add a check here to make sure there is a country, or make sure `isPostcode` correctly handles the absence. Probably better to check here and only validate when country is also set. We can revalidate later?

Reply:
Good catch. I used the second suggestion and added a check for the country.